db/migrate/20200206163659_add_migrated_at_to_complete_moabs.rb

```diff
@@ -0,0 +1,5 @@
+class AddMigratedAtToCompleteMoabs < ActiveRecord::Migration[6.0]
+  def change
+    add_column :complete_moabs, :from_moab_storage_root_id, :bigint
```

Comment:
can't remember off-hand what the syntax is, esp since this is a second FK relationship to `moab_storage_roots`, but could we add a DB level foreign key constraint to this column?

Reply:
I'll  test a  FK.  I was  getting an error with it originally  but I think that was  fixed with the `required: false` so a FK is likely a good idea.